packages/react-vapor/src/components/blankSlate/BlankSlateWithTable.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import {IBlankSlateProps} from './BlankSlate';
+
+export interface IBlankSlateWithTableProps extends IBlankSlateWithTableConfig, IBlankSlateProps {}
+
+export interface IBlankSlateWithTableConfig {
+    numberOfColumn?: number;
+}
+
+export const blankSlateWithTable = <P extends IBlankSlateProps>(Component: React.ComponentType<P>) => {
+    class ComponentWithTable extends React.PureComponent<Partial<IBlankSlateWithTableProps>> {
```

Comment:
If any prop from `IBlankSlateProps` is required, the `Partial` here will make them optional and this could cause errors.
```suggestion
    class ComponentWithTable extends React.PureComponent<IBlankSlateWithTableProps> {
```